init-job/base/argo-workflows/initialize-schema.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: initialize-schema
+  annotations:
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: thoth
+    component: initialize-schema
+spec:
+  templates:
+    - name: initialize-schema
+      inputs:
+        parameters:
```

Comment:
we dont need these parameters, we need to make sure that the imagestream `init-job` is created where the workflow runs. I think that will be middletier?

Reply:
yes exactly

Reply:
```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: stage-thoth-init-job
spec:
  destination:
    namespace: thoth-middletier-stage
    server: "https://api.thoth01.lab.upshift.rdu2.redhat.com:6443"
  source:
    path: adviser/overlays/stage
    repoURL: "https://github.com/thoth-station/thoth-application.git"
    targetRevision: master
  project: thoth
  syncPolicy:
    automated:
      automated:
        prune: false
        selfHeal: false
      selfHeal: true
```